csharp/test/Glacier2/router/Client.cs

```diff
@@ -55,9 +55,8 @@ public override async Task RunAsync(string[] args)
             {
                 Console.Out.Write("getting the session timeout... ");
                 Console.Out.Flush();
-                long sessionTimeout = router.GetSessionTimeout();
-                long acmTimeout = router.GetACMTimeout();
-                Assert(sessionTimeout == 30 && acmTimeout == 30);
+                long timeout = router.GetSessionTimeout();
```

Comment:
Why not use the ACM timeout instead which is the preferred way now to keep alive the session? We'll eventually remove the old session timeout in future Ice 3.x versions.